doc/developer/index.adoc

```diff
@@ -42,7 +42,10 @@ RDF4J uses a git branching model where feature development takes place on branch
 These branches should have the prefix `features/` to indicate they are either
 not complete or have not been scheduled for the next minor release.
 
-Every issue, no matter how small, gets its own branch while under development. Issue branch names are always prefixed with `issues/`, followed by the issue number in the GitHub issue tracker, followed by one or two dash-separated keywords for the issue.
+Every issue, no matter how small, gets its own branch while under development.
+The milestone label of the issue is set to the *planned* release version for the
```

Comment:
It can't be a contributor as they do not have permission to edit the milestone field, AFAIK. So it would have to be committers or the release manager.